doc/conf.py

```diff
@@ -78,9 +78,5 @@
     '**/.tox',
 ]
 
-intersphinx_mapping = {
```

Comment:
What is the problem with this? Removing this breaks all :doc: directives in my local build.

Reply:
gkunz why would the global doc build require an intersphinx mapping to itself? It worked here and I'm very surprised it could fail in your env.

Reply:
@gkunz could u help me to reproduce your failure?

$ rm -rf _build .tox ; LC_ALL=C tox
...
  docs: commands succeeded
  congratulations :)

and I will see all std:doc in _build/html/objects.inv

Reply:
@collivier because the doc role is often used in conjunction with a defined intersphinx mapping. See https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role

In our docs, see for example https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter01.rst#definitionsterminologyabbreviations

The build does not fail, but the links are not getting generated.

Reply:
for sure ref is not. checking.

Reply:
Yes you're right for :doc: - and I confirm :ref: fallback to local labels. We have to loop to ourselves here.

BYW we will have to remove all :doc: to common in RM and RA1 soon.

Reply:
@gkunz done